### ISSUE TITLE
Music App Tries to open in an Infinite Loop

### ISSUE THREAD

**Issue by mig8447** [tombonez/noTunes#48]
I recently I'm having an issue where Music tries to open, it gets killed by noTunes and it continues trying to open  in a sort of an infinite loop, stealing the focus of whatever app you're in which may even prevent my from stopping noTunes. Has anyone experienced the same? (Sonoma 14.5)

**Comment by tombonez** [comment]
Hi @mig8447 👋🏻

I'd recommend finding out what is launching Apple Music and why. Any ideas? As something attempting to open up Apple Music over and over seems incredibly strange.

It's a long shot, but have you possibly used the [Set replacement for iTunes / Apple Music](https://github.com/tombonez/noTunes?tab=readme-ov-file#set-replacement-for-itunes--apple-music) feature have you and set Apple Music as the replacement, causing an endless loop?

Cheers,
Tom 🙂

**Comment by mig8447** [comment]
Thanks for the quick response. This happens after a zoom.us call ends and my Bluetooth headset is connected, then the Music app opens on its own and this infinite loop happens.

I just checked and the replacement is not enabled.

**Comment by tombonez** [comment]
Very strange. What happens when noTunes isn't open (or disabled) when this behaviour triggers, does Apple Music keep opening when quitting it too? This appears to be a bizarre Zoom and/or Bluetooth headset issue that might be possible to work around.

**Comment by mig8447** [comment]
No, it just opens once, and stops there, but then I have to close it manually

**Comment by mig8447** [comment]
If it helps, headset is a pair of Bose 700 UC connected through the USB dongle

**Comment by lumpenpancho** [comment]
Facing the same issue.

**Comment by evgenytaychinov-afk** [comment]
Facing same
